lib/cisco_node_utils/cmd_ref/acl.yaml

```diff
@@ -0,0 +1,33 @@
+# Command Reference Common ACL
+#
+# For documentation please see:
+#   - README_YAML.md
+#
+---
+_template:
+  config_get: 'show run aclmgr'
+  config_get_token: '/^<afi> access-list <acl_name>$/'
+  config_set: '<afi> access-list <acl_name>'
+
+ace:
+  config_get_token_append: '/^(<seqno>) .*$'
+  config_set_append:
+      '<state> <seqno> <action> <proto> <src_addr> <src_port> <dst_addr> <dst_port> <option_format>'
+
+acl:
+  config_get_token: '/^<afi> access-list (\S+)$/'
+  config_set: '<state> <afi> access-list <acl_name>'
+                      
+all_ace:
+  multiple:
+  config_get_token_append: '/^(\d+) .*$/'
+
+all_acl:
+  multiple:
+  config_get_token: '/^<afi> access-list (\S+)$/'
+
+stats_perentry:
```

Comment:
*nit* Here and elsewhere I would name this `stats_per_entry`.  Bit easier to read IMO.